metabase/metabase
# Enable zoom drill on questions based on questions

Context https://metaboat.slack.com/archives/C01LQQ2UW03/p1722478398530239

How to verify:
- New -> Question -> Orders -> Save as Q1
- New -> Question -> Q1 -> Visualize
- You can click on any cell and then "View details" to open the object detail view
- Note that for FKs it works a bit differently but it's the same for tables
